tests/ignite/metrics/test_running_average.py

```diff
@@ -187,6 +187,9 @@ def assert_equal_running_avg_output_values(engine):
 
     trainer.run(data, max_epochs=3)
 
+    running_avg_acc[:] = [None]
+    trainer.run([1] * n_iters, max_epochs=3)
```

Comment:
and why not reuse previous `data`?
```suggestion
    trainer.run(data, max_epochs=3)
```